include/server.h

```diff
@@ -88,6 +90,8 @@ struct server {
     struct wl_listener new_xdg_surface;
     struct wl_listener new_layer_shell_surface;
     struct wl_listener new_pointer_constraint;
+	  struct wl_listener output_mgr_apply;
```

Comment:
indentation seems wrong

Reply:
doesn't look lite this in my code, could be a tabs vs 4 spaces problem here same, possibly the same problem in the later one that looks wonky